packages/core/test/unit/html/NodeProcessor.data.js

```diff
@@ -368,36 +301,26 @@ module.exports.PROCESS_DROPDOWN_HEADER_EXPECTED = `
 </dropdown>
 `;
 
-// TODO deprecate text attribute of dropdown
-module.exports.PROCESS_DROPDOWN_TEXT_ATTR = `
-<dropdown text="**Lorem ipsum dolor sit amet**">
-  Text attribute should be inserted as header slot and deleted.
-</dropdown>
-`;
-
-// TODO deprecate text attribute of dropdown
 module.exports.PROCESS_DROPDOWN_TEXT_ATTR_EXPECTED = `
 <dropdown><template #header><strong>Lorem ipsum dolor sit amet</strong></template>
   Text attribute should be inserted as header slot and deleted.
 </dropdown>
 `;
```

Comment:
This can be removed as it is no longer used :)

Reply:
Ah sorry I missed that out! Have removed these 😅